### PR TITLE
Qgis dimensions

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Add support for dimensions_filters (apply dimensions through OGC filters).
Relate https://github.com/camptocamp/c2cgeoportal/pull/3840

### CODE REVIEW

File: contribs/gmf/src/directives/editfeature.js [hunk 1]
```diff
@@ -703,9 +717,11 @@ gmf.EditfeatureController.prototype.delete = function() {
         // (2) Reset selected feature
         this.cancel();
       },
-      () => {
+      (response) => {
         this.showServerError = true;
         this.pending = false;
+        this.serverErrorType =  `error type : ${response.data['error_type']}`;
+        this.serverErrorMessage = `error message : ${response.data['message']}`;
```
Comment (sbrunner) [inline contribs/gmf/src/directives/editfeature.js]: No space before ':' in english...

File: contribs/gmf/src/directives/editfeature.js [hunk 2]
```diff
@@ -612,9 +624,11 @@ gmf.EditfeatureController.prototype.save = function() {
       this.pending = false;
       this.handleEditFeature_(response);
     },
-    () => {
+    (response) => {
       this.showServerError = true;
       this.pending = false;
+      this.serverErrorType =  `error type : ${response.data['error_type']}`;
+      this.serverErrorMessage = `error message : ${response.data['message']}`;
```
Comment (sbrunner) [inline contribs/gmf/src/directives/editfeature.js]: No space before ':' in english...

File: contribs/gmf/src/directives/partials/editfeature.html [hunk 3]
```diff
@@ -128,6 +128,7 @@ <h4 class="modal-title">
   </div>
   <ngeo-modal ng-model="efCtrl.showServerError" ngeo-modal-destroy-content-on-hide="true">
     <div class="modal-header">{{'Server error.' | translate}}</div>
-    <div class="modal-body">{{'Unexpected server error.' | translate}}</div>
+    <div class="modal-body">{{$parent.efCtrl.serverErrorType}}<br>
+    {{$parent.efCtrl.serverErrorMessage || ('Unexpected server error.' | translate)}}</div>
```
Comment (sbrunner) [inline contribs/gmf/src/directives/partials/editfeature.html]: I never see this syntax is correct?

**Comment by arnaud-morvan** [comment]
Your comments are about showing postgresql integrity errors on client side, nothing to do with QGIS.
These commits are here because we use that branch with ADL, we may remove them before merge.

**Comment by arnaud-morvan** [comment]
Merged in 2.3: #4223

**Pull request closed without merging**